index.js

```diff
@@ -26,16 +25,32 @@ export {
 };
 
 let loaded;
-export async function loadOnce(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function loadOnce(options) {
   if(loaded) {
     return loaded;
   }
 
   loaded = true;
-  return load(mediatorUrl);
+  return load(options);
 }
 
-export async function load(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function load(options = {
+  mediatorOrigin: DEFAULT_MEDIATOR_ORIGIN
```

Comment:
Should have some jsdocs and README docs for `loadOnce` and `load` and the options.

Reply:
Indeed, but would be too low on my priority list for me to take that action at this time. PRs welcome :)